src/services/DiscourseService.ts

```diff
@@ -148,4 +153,28 @@ export class DiscourseService {
       return null
     }
   }
+
+  static commentUpdatedProposal(updatedProposal: ProposalAttributes) {
+    inBackground(async () => {
+      // TODO: votes are not necessary in all cases, they could be fetched only when needed
```

Comment:
When is it not necessary?

Reply:
Oh ok I see the check is in getUpdateMessage. Maybe I'd move the `// TODO:` to that file (It is asking for a refactor)